File: LegacyExample/ChildViewController.m
//
//  ChildViewController.m
//  LegacyExample
//
//  Created by Jeff Kelley on 7/19/18.
//  Copyright © 2018 Detroit Labs. All rights reserved.
//

#import "ChildViewController.h"

@interface ChildViewController ()

@property (weak, nonatomic) IBOutlet UIScrollView *scrollView;
@property (weak, nonatomic) IBOutlet UIView *contentView;

@end

@implementation ChildViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    self.view.translatesAutoresizingMaskIntoConstraints = NO;
    self.scrollView.translatesAutoresizingMaskIntoConstraints = NO;
    self.contentView.translatesAutoresizingMaskIntoConstraints = NO;
}

@end

File: Example/ChildViewController.swift
//
//  ChildViewController.swift
//  SheetPresentation
//
//  Created by Jeff Kelley on 7/17/18.
//  Copyright © 2020 Detroit Labs, LLC. All rights reserved.
//

import UIKit

class ChildViewController: UIViewController {

    @IBOutlet weak var scrollView: UIScrollView?
    @IBOutlet weak var contentView: UIView?
    @IBOutlet weak var stackView: UIStackView?

    override func viewDidLoad() {
        super.viewDidLoad()

        // Uncomment to see how a preferred content size is handled.
        // preferredContentSize = CGSize(width: 200, height: 500)
    }

    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
        return .all
    }

    override func traitCollectionDidChange(
        _ previousTraitCollection: UITraitCollection?
    ) {
        super.traitCollectionDidChange(previousTraitCollection)
    }

}
